test/Twig/Tests/Profiler/Dumper/AbstractTest.php

```diff
@@ -32,13 +32,14 @@ protected function getProfile()
 
         $embedded = clone $embedded;
         $index->addProfile($embedded);
-        $a = range(1, 1000);
```

Comment:
This is effectively a nonop and will be removed by the compiler removing its use (delay)